spec/Section 3 -- Type System.md

```diff
@@ -919,7 +919,7 @@ of rules must be adhered to by every Object type in a GraphQL schema.
       3. The argument must accept a type where {IsInputType(argumentType)}
          returns {true}.
       4. If argument type is Non-Null and a default value is not defined:
-         - The `@deprecated` directive must not be applied to this argument.
+         1. The `@deprecated` directive must not be applied to this argument.
```

Comment:
Would the unordered list be preferred in this case?

Reply:
IIRC, it would be the only unordered item in any of our equivalent algorithms, so I changed it for consistency.